app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -176,13 +242,21 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             @Override
             protected void doTint(Context themedContext) {
                 toolbar.getMenu().findItem(R.id.sort_items)
-                        .setIcon(AppCompatDrawableManager.get().getDrawable(themedContext, R.drawable.ic_sort));
+                        .setIcon(R.drawable.ic_sort);
```

Comment:
Are you sure that this works? The icon color should change when scrolling in the light theme. (White when on the blurred background, black when on the toolbar).

Reply:
I had to make this change because I one of your recents commits removed the black and white icons that are used here. I was going to create a separate issue later.

Reply:
On my device, the AppCompatDrawableManager performs the color changes automatically. The images reference an attr resource internally.

Reply:
That's not the problem. The problem is that the ide cannot find the icons. One of your recent commits removed the attr referfence.

Reply:
The original/removed line here does not mention an attr reference

Reply:
@ByteHamster Ok, fixed.